src/Game.js

```diff
@@ -0,0 +1,48 @@
+function Game() {
+  this._total_score = 0;
+  this._frame_score = 0;
+  this.is_a_strike = false;
+  this.is_a_spare = false;
+  this.frame_count = 0;
+};
+
+Game.prototype.total_score = function(roll1, roll2) {
```

Comment:
Small thing which will seem incredibly pedantic! In JS, they use camelCase instead of underscores_for_functions

Reply:
good point!